tls-codec/src/hpke.rs

```diff
@@ -0,0 +1,18 @@
+//! TLS Codec for HPKE primitives
+
+use super::*;
+use hpke_rs::HPKEPublicKey;
+
+impl Serialize for HPKEPublicKey {
```

Comment:
Isn't the purpose of this macro that we can just derive `TlsSerialize` for `HPKEPublicKey`? Or am I missing something?

Reply:
`HPKEPublicKey` is define in the HPKE crate that doesn't know about this (and doesn't really care). So we can't derive but have to implement it manually.